[magnum] Build failure

Package: magnum[core,gl,text,texturetools]:arm64-osx -> 2020.06#14

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 15.0.0.15000040
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: db0473513 2023-09-27 (31 hours ago)

**To Reproduce**

`vcpkg install --enforce-port-checks --allow-unsupported`
**Failure logs**

```
-- Using cached mosra-magnum-v2020.06.tar.gz.
-- Cleaning sources at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/mosra-magnum-v2020.06.tar.gz
-- Applying patch 002-sdl-includes.patch
-- Applying patch 003-fix-FindGLFW.patch
-- Applying patch 004-fix-FindOpenAL.patch
-- Applying patch 005-fix-find-sdl2.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean
CMake Warning at ports/magnum/portfile.cmake:53 (message):
  Features anyshaderconverter, shadertools, shaderconverter and vk-info are
  not avaliable when building non-head version.
Call Stack (most recent call first):
  scripts/ports.cmake:147 (include)


-- Found external ninja('1.11.1').
-- Configuring arm64-osx
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.27.1-osx/cmake-3.27.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/magnum/portfile.cmake:86 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/install-arm64-osx-dbg-out.log</summary>

```
Change Dir: '/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg'

Run Build Command(s): /opt/homebrew/bin/ninja -v -v -j11 install
[1/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/MagnumExternal/OpenGL/GL/CMakeFiles/MagnumFlextGLObjects.dir/flextGL.cpp.o -MF src/MagnumExternal/OpenGL/GL/CMakeFiles/MagnumFlextGLObjects.dir/flextGL.cpp.o.d -o src/MagnumExternal/OpenGL/GL/CMakeFiles/MagnumFlextGLObjects.dir/flextGL.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/MagnumExternal/OpenGL/GL/flextGL.cpp
[2/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/CMakeFiles/MagnumMathObjects.dir/Math/Packing.cpp.o -MF src/Magnum/CMakeFiles/MagnumMathObjects.dir/Math/Packing.cpp.o.d -o src/Magnum/CMakeFiles/MagnumMathObjects.dir/Math/Packing.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Packing.cpp
[3/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/CMakeFiles/MagnumObjects.dir/Sampler.cpp.o -MF src/Magnum/CMakeFiles/MagnumObjects.dir/Sampler.cpp.o.d -o src/Magnum/CMakeFiles/MagnumObjects.dir/Sampler.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Sampler.cpp
...
Skipped 88 lines
...
[20/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/CMakeFiles/Magnum.dir/VertexFormat.cpp.o -MF src/Magnum/CMakeFiles/Magnum.dir/VertexFormat.cpp.o.d -o src/Magnum/CMakeFiles/Magnum.dir/VertexFormat.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/VertexFormat.cpp
[21/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/CMakeFiles/Magnum.dir/Animation/Player.cpp.o -MF src/Magnum/CMakeFiles/Magnum.dir/Animation/Player.cpp.o.d -o src/Magnum/CMakeFiles/Magnum.dir/Animation/Player.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Animation/Player.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Animation/Player.cpp:26:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Animation/Player.hpp:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Animation/Player.h:35:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Animation/Track.h:36:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:153:32: warning: definition of implicit copy assignment operator for 'Range<1, float>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Range(const Range<dimensions, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Animation/Player.h:289:49: note: in implicit copy assignment operator for 'Magnum::Math::Range<1, float>' first required here
        Player<T, K>& operator=(Player<T, K>&&) noexcept;
                                                ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Animation/Player.hpp:81:56: note: in defaulted move assignment operator for 'Magnum::Animation::Player<Float, Float>' first required here
template<class T, class K> Player<T, K>& Player<T, K>::operator=(Player<T, K>&&) noexcept = default;
                                                       ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Animation/Player.cpp:56:34: note: in instantiation of member function 'Magnum::Animation::Player<float>::operator=' requested here
template class MAGNUM_EXPORT_HPP Player<Float, Float>;
                                 ^
1 warning generated.
[22/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/AbstractQuery.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/AbstractQuery.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/AbstractQuery.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractQuery.cpp
[23/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Buffer.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Buffer.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Buffer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Buffer.cpp
FAILED: src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Buffer.cpp.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Buffer.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Buffer.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Buffer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Buffer.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Buffer.cpp:26:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Buffer.h:36:
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/Corrade/Utility/StlForwardTuple.h:58:10: error: '__tuple' file not found, did you mean 'tuple'?
#include <__tuple>
         ^~~~~~~~~
         <tuple>
1 error generated.
[24/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/AbstractObject.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/AbstractObject.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/AbstractObject.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractObject.cpp
[25/80] : && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.27.1-osx/cmake-3.27.1-macos-universal/CMake.app/Contents/bin/cmake -E rm -f Debug/lib/libMagnum-d.a && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar qc Debug/lib/libMagnum-d.a  src/Magnum/CMakeFiles/MagnumMathObjects.dir/Math/Angle.cpp.o src/Magnum/CMakeFiles/MagnumMathObjects.dir/Math/Color.cpp.o src/Magnum/CMakeFiles/MagnumMathObjects.dir/Math/Half.cpp.o src/Magnum/CMakeFiles/MagnumMathObjects.dir/Math/Functions.cpp.o src/Magnum/CMakeFiles/MagnumMathObjects.dir/Math/Packing.cpp.o src/Magnum/CMakeFiles/MagnumMathObjects.dir/Math/instantiation.cpp.o src/Magnum/CMakeFiles/MagnumObjects.dir/FileCallback.cpp.o src/Magnum/CMakeFiles/MagnumObjects.dir/PixelStorage.cpp.o src/Magnum/CMakeFiles/MagnumObjects.dir/Resource.cpp.o src/Magnum/CMakeFiles/MagnumObjects.dir/Sampler.cpp.o src/Magnum/CMakeFiles/MagnumObjects.dir/Timeline.cpp.o src/Magnum/CMakeFiles/Magnum.dir/Math/PackingBatch.cpp.o src/Magnum/CMakeFiles/Magnum.dir/Image.cpp.o src/Magnum/CMakeFiles/Magnum.dir/ImageView.cpp.o src/Magnum/CMakeFiles/Magnum.dir/Mesh.cpp.o src/Magnum/CMakeFiles/Magnum.dir/PixelFormat.cpp.o src/Magnum/CMakeFiles/Magnum.dir/VertexFormat.cpp.o src/Magnum/CMakeFiles/Magnum.dir/Animation/Player.cpp.o src/Magnum/CMakeFiles/Magnum.dir/Animation/Interpolation.cpp.o && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib Debug/lib/libMagnum-d.a && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.27.1-osx/cmake-3.27.1-macos-universal/CMake.app/Contents/bin/cmake -E touch Debug/lib/libMagnum-d.a && :
[26/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/TextureFormat.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/TextureFormat.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/TextureFormat.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/TextureFormat.cpp
[27/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Framebuffer.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Framebuffer.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Framebuffer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.cpp
FAILED: src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Framebuffer.cpp.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Framebuffer.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Framebuffer.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Framebuffer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.cpp:26:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.h:35:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:36:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:153:32: warning: definition of implicit copy assignment operator for 'Range<2, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Range(const Range<dimensions, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Math.h:102:23: note: in implicit copy assignment operator for 'Magnum::Math::Range<2, int>' first required here
template<class> class Range2D;
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:751:29: note: in implicit move assignment operator for 'Magnum::Math::Range2D<int>' first required here
            other._viewport = {};
                            ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.cpp:26:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.h:35:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:36:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Functions.h:38:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector.h:230:32: warning: definition of implicit copy assignment operator for 'Vector<2, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Vector(const Vector<size, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector2.h:64:25: note: in implicit copy assignment operator for 'Magnum::Math::Vector<2, int>' first required here
template<class T> class Vector2: public Vector<2, T> {
                        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:64:49: note: in implicit copy assignment operator for 'Magnum::Math::Vector2<int>' first required here
template<UnsignedInt dimensions, class T> class Range {
                                                ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Math.h:102:23: note: in implicit copy assignment operator for 'Magnum::Math::Range<2, int>' first required here
template<class> class Range2D;
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:751:29: note: in implicit move assignment operator for 'Magnum::Math::Range2D<int>' first required here
            other._viewport = {};
                            ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.cpp:26:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.h:35:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:36:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Functions.h:38:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector.h:230:32: warning: definition of implicit copy assignment operator for 'Vector<3, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Vector(const Vector<size, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector3.h:64:25: note: in implicit copy assignment operator for 'Magnum::Math::Vector<3, int>' first required here
template<class T> class Vector3: public Vector<3, T> {
                        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/PixelStorage.h:119:19: note: in implicit copy assignment operator for 'Magnum::Math::Vector3<int>' first required here
            _skip = skip;
                  ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.cpp:38:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/BufferImage.h:36:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Buffer.h:36:
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/Corrade/Utility/StlForwardTuple.h:58:10: error: '__tuple' file not found, did you mean 'tuple'?
#include <__tuple>
         ^~~~~~~~~
         <tuple>
3 warnings and 1 error generated.
[28/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Context.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Context.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Context.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Context.cpp
FAILED: src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Context.cpp.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Context.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Context.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Context.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Context.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Context.cpp:37:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:36:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:153:32: warning: definition of implicit copy assignment operator for 'Range<2, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Range(const Range<dimensions, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Math.h:102:23: note: in implicit copy assignment operator for 'Magnum::Math::Range<2, int>' first required here
template<class> class Range2D;
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:751:29: note: in implicit move assignment operator for 'Magnum::Math::Range2D<int>' first required here
            other._viewport = {};
                            ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Context.cpp:37:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:36:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Functions.h:38:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector.h:230:32: warning: definition of implicit copy assignment operator for 'Vector<2, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Vector(const Vector<size, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector2.h:64:25: note: in implicit copy assignment operator for 'Magnum::Math::Vector<2, int>' first required here
template<class T> class Vector2: public Vector<2, T> {
                        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:64:49: note: in implicit copy assignment operator for 'Magnum::Math::Vector2<int>' first required here
template<UnsignedInt dimensions, class T> class Range {
                                                ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Math.h:102:23: note: in implicit copy assignment operator for 'Magnum::Math::Range<2, int>' first required here
template<class> class Range2D;
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:751:29: note: in implicit move assignment operator for 'Magnum::Math::Range2D<int>' first required here
            other._viewport = {};
                            ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Context.cpp:40:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Buffer.h:36:
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/Corrade/Utility/StlForwardTuple.h:58:10: error: '__tuple' file not found, did you mean 'tuple'?
#include <__tuple>
         ^~~~~~~~~
         <tuple>
2 warnings and 1 error generated.
[29/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Renderbuffer.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Renderbuffer.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Renderbuffer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Renderbuffer.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Renderbuffer.cpp:34:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Implementation/FramebufferState.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.h:35:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:36:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:153:32: warning: definition of implicit copy assignment operator for 'Range<2, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Range(const Range<dimensions, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Math.h:102:23: note: in implicit copy assignment operator for 'Magnum::Math::Range<2, int>' first required here
template<class> class Range2D;
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:751:29: note: in implicit move assignment operator for 'Magnum::Math::Range2D<int>' first required here
            other._viewport = {};
                            ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Renderbuffer.cpp:34:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Implementation/FramebufferState.h:31:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Framebuffer.h:35:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:36:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Functions.h:38:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector.h:230:32: warning: definition of implicit copy assignment operator for 'Vector<2, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Vector(const Vector<size, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector2.h:64:25: note: in implicit copy assignment operator for 'Magnum::Math::Vector<2, int>' first required here
template<class T> class Vector2: public Vector<2, T> {
                        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:64:49: note: in implicit copy assignment operator for 'Magnum::Math::Vector2<int>' first required here
template<UnsignedInt dimensions, class T> class Range {
                                                ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Math.h:102:23: note: in implicit copy assignment operator for 'Magnum::Math::Range<2, int>' first required here
template<class> class Range2D;
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:751:29: note: in implicit move assignment operator for 'Magnum::Math::Range2D<int>' first required here
            other._viewport = {};
                            ^
2 warnings generated.
[30/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/DefaultFramebuffer.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/DefaultFramebuffer.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/DefaultFramebuffer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/DefaultFramebuffer.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/DefaultFramebuffer.cpp:26:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/DefaultFramebuffer.h:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:36:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:153:32: warning: definition of implicit copy assignment operator for 'Range<2, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Range(const Range<dimensions, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Math.h:102:23: note: in implicit copy assignment operator for 'Magnum::Math::Range<2, int>' first required here
template<class> class Range2D;
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:751:29: note: in implicit move assignment operator for 'Magnum::Math::Range2D<int>' first required here
            other._viewport = {};
                            ^
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/DefaultFramebuffer.cpp:26:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/DefaultFramebuffer.h:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:36:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Functions.h:38:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector.h:230:32: warning: definition of implicit copy assignment operator for 'Vector<2, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Vector(const Vector<size, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector2.h:64:25: note: in implicit copy assignment operator for 'Magnum::Math::Vector<2, int>' first required here
template<class T> class Vector2: public Vector<2, T> {
                        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:64:49: note: in implicit copy assignment operator for 'Magnum::Math::Vector2<int>' first required here
template<UnsignedInt dimensions, class T> class Range {
                                                ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Math.h:102:23: note: in implicit copy assignment operator for 'Magnum::Math::Range<2, int>' first required here
template<class> class Range2D;
                      ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/AbstractFramebuffer.h:751:29: note: in implicit move assignment operator for 'Magnum::Math::Range2D<int>' first required here
            other._viewport = {};
                            ^
2 warnings generated.
[31/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Texture.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Texture.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Texture.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Texture.cpp
FAILED: src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Texture.cpp.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Texture.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Texture.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Texture.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Texture.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Texture.cpp:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Implementation/TextureState.h:31:
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/Corrade/Utility/StlForwardTuple.h:58:10: error: '__tuple' file not found, did you mean 'tuple'?
#include <__tuple>
         ^~~~~~~~~
         <tuple>
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Texture.cpp:26:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Texture.h:37:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector3.h:32:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector2.h:32:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector.h:230:32: warning: definition of implicit copy assignment operator for 'Vector<3, int>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Vector(const Vector<size, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Vector3.h:64:25: note: in implicit copy assignment operator for 'Magnum::Math::Vector<3, int>' first required here
template<class T> class Vector3: public Vector<3, T> {
                        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/PixelStorage.h:119:19: note: in implicit copy assignment operator for 'Magnum::Math::Vector3<int>' first required here
            _skip = skip;
                  ^
1 warning and 1 error generated.
[32/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Renderer.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Renderer.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Renderer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Renderer.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Renderer.cpp:31:
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/Math/Range.h:153:32: warning: definition of implicit copy assignment operator for 'Range<1, float>' is deprecated because it has a user-declared copy constructor [-Wdeprecated-copy]
        constexpr /*implicit*/ Range(const Range<dimensions, T>&) noexcept = default;
                               ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Renderer.cpp:44:15: note: in implicit copy assignment operator for 'Magnum::Math::Range<1, float>' first required here
        value = state.lineWidthRangeImplementation();
              ^
1 warning generated.
[33/80] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DCORRADE_IS_DEBUG_BUILD -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/arm64-osx-dbg/src -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.0.sdk -mmacosx-version-min=13.6 -fPIC -std=c++11 -Wall -Wextra -Wold-style-cast -Winit-self -Werror=return-type -Wmissing-declarations -Wpedantic -fvisibility=hidden -fvisibility-inlines-hidden -Wmissing-prototypes -Wno-shorten-64-to-32 -MD -MT src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Shader.cpp.o -MF src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Shader.cpp.o.d -o src/Magnum/GL/CMakeFiles/MagnumGLObjects.dir/Shader.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/magnum/src/v2020.06-4ccec3ac49.clean/src/Magnum/GL/Shader.cpp
ninja: build stopped: subcommand failed.
```
</details>**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "magnum",
      "default-features": false,
      "features": [
        "text"
      ]
    }
  ],
  "$overrides": [
    {
      "name": "llvm",
      "version": "14.0.6#1"
    }
  ],
  "$builtin-baseline": "e68bd1cc40a13fd47d999ecff333e8b809b2c720"
}

```
</details>

--- a/ports/corrade/clang-16.patch
+++ b/ports/corrade/clang-16.patch
@@ -1,0 +1,32 @@
+diff --git a/src/Corrade/Utility/StlForwardArray.h b/src/Corrade/Utility/StlForwardArray.h
+index 0e78d61..363fafb 100644
+--- a/src/Corrade/Utility/StlForwardArray.h
++++ b/src/Corrade/Utility/StlForwardArray.h
+@@ -55,7 +55,11 @@ possible or is unknown is equivalent to @cpp #include <array> @ce.
+ 
+ #ifdef CORRADE_TARGET_LIBCXX
+ /* https://github.com/llvm-mirror/libcxx/blob/73d2eccc78ac83d5947243c4d26a53f668b4f432/include/__tuple#L223 */
++#if _LIBCPP_VERSION < 160000
+ #include <__tuple>
++#else
++#include <__fwd/array.h>
++#endif
+ #elif defined(CORRADE_TARGET_DINKUMWARE)
+ /* MSVC has it defined next to std::pair */
+ #include <utility>
+diff --git a/src/Corrade/Utility/StlForwardTuple.h b/src/Corrade/Utility/StlForwardTuple.h
+index 5701bc0..3292ec8 100644
+--- a/src/Corrade/Utility/StlForwardTuple.h
++++ b/src/Corrade/Utility/StlForwardTuple.h
+@@ -55,7 +55,11 @@ is equivalent to @cpp #include <tuple> @ce.
+ 
+ #ifdef CORRADE_TARGET_LIBCXX
+ /* https://github.com/llvm-mirror/libcxx/blob/73d2eccc78ac83d5947243c4d26a53f668b4f432/include/__tuple#L163 */
++#if _LIBCPP_VERSION < 160000
+ #include <__tuple>
++#else
++#include <__fwd/tuple.h>
++#endif
+ #elif defined(CORRADE_TARGET_LIBSTDCXX)
+ /* https://github.com/gcc-mirror/gcc/blob/c014d57d57a03e6061a57fa8534e90979567392b/libstdc%2B%2B-v3/include/std/type_traits#L2465-L2466 */
+ #include <type_traits>

--- a/ports/corrade/portfile.cmake
+++ b/ports/corrade/portfile.cmake
@@ -7,6 +7,7 @@ vcpkg_from_github(
     PATCHES
         fix-vs2019.patch
         build-corrade-rc-always.patch
+        clang-16.patch
 )
 
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" BUILD_STATIC)

--- a/ports/corrade/vcpkg.json
+++ b/ports/corrade/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "corrade",
   "version-string": "2020.06",
-  "port-version": 6,
+  "port-version": 7,
   "description": "C++11/C++14 multiplatform utility library.",
   "homepage": "https://magnum.graphics/corrade/",
   "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1770,7 +1770,7 @@
     },
     "corrade": {
       "baseline": "2020.06",
-      "port-version": 6
+      "port-version": 7
     },
     "cpp-async": {
       "baseline": "1.0.1",

--- a/versions/c-/corrade.json
+++ b/versions/c-/corrade.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "8630a6d00fec6c7462af2e03bd0e1946f6812069",
+      "version-string": "2020.06",
+      "port-version": 7
+    },
+    {
       "git-tree": "1a278fdbd46715f454e366df7d06410ae2fc9747",
       "version-string": "2020.06",
       "port-version": 6

